src/components/Form.tsx

```diff
@@ -90,9 +78,22 @@ export default function Form() {
         <CreateLabel description="Wybierz godzinę:">
           <CreateInput type="time" value={time} changeState={setTime} />
         </CreateLabel>
-        <CreateInput type="submit" value="Wyślij" className="m-auto" />
+        <button type="submit" className="m-auto">
+          Wyślij
+        </button>
       </form>
-      {isSend && <FormSummary/>}
+      {isSend && (
+        <FormSummary
```

Comment:
These component props should be a single object, like formData or sth. Notice that it can be passed via context, but don’t move it to contwst right now